bin/fin

```diff
@@ -3417,18 +3558,34 @@ case "$1" in
 		shift
 		restart
 		;;
-	ps)
-		_ps
+	project)
+		shift
+		case "$1" in
+			status)
```

Comment:
Since "project" is a global management command, I don't think we should have "status" (which is local, project level) at this level.